devicon.json

```
@@ -335,7 +335,10 @@
     },
     {
         "name": "apache",
-        "altnames": [],
+        "altnames": [
+            "The Apache Software Foundation™",
+            "The Apache Software Foundation"
+        ],
```

Maybe the one with trademark character is unnecessary, isn't it @Snailedlt? 🤔
```suggestion
        "altnames": [
            "The Apache Software Foundation"
        ],
```

---

Hmm, yeah we only need one of them.  I doubt anyone would use tm in a search, but it is supported, so either one is fine, but no need for both since it's only used for searching :)